source/jormungandr/jormungandr/instance_manager.py

```diff
@@ -155,7 +155,8 @@ def initialization(self):
 
         # we fetch the krakens metadata first
         # not on the ping thread to always have the data available (for the tests for example)
-        self.init_kraken_instances()
+        if app.config.get('INIT_KRAKEN_INSTANCES', False):
```

Comment:
you want to make the init when the parameter is false ?
```suggestion
        if app.config.get('INIT_KRAKEN_INSTANCES', True):
```

Reply:
`False` here is the default value :wink: